### DISCUSSION TITLE
In the "One to Many" interface config, the "Enable searching & filtering" toggle doesn't seem to do anything. Am I missing something?

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#17121] [Archive]
In the "One to Many" interface config, the "Enable searching & filtering" toggle doesn't seem to do anything.

**Am I missing something?**

# The Toggle

<img width="271" alt="image" src="https://user-images.githubusercontent.com/26592486/212136210-f3696f0f-3ce9-4029-9d52-9a5ac7c51b7e.png">

# Field With "Enable searching & filtering" Disabled

<img width="766" alt="image" src="https://user-images.githubusercontent.com/26592486/212136373-b0ebed9d-75ba-477a-818c-3fdc69ec67c3.png">

# Field With "Enable searching & filtering" Enabled

<img width="766" alt="image" src="https://user-images.githubusercontent.com/26592486/212136664-1bd21523-b70c-400a-a1f7-64ce537b58ee.png">

As you can see, there is zero difference.

And when clicking the upper right "Add Existing" button, there is no difference in the panel that pops up either.

# "Add Existing" Panel With "Enable searching & filtering" Disabled

<img width="796" alt="image" src="https://user-images.githubusercontent.com/26592486/212137190-2bbeb5f8-8ceb-4154-9553-d6b1680189b2.png">

# "Add Existing" Panel With "Enable searching & filtering" Enabled

<img width="796" alt="image" src="https://user-images.githubusercontent.com/26592486/212137367-6f000f2d-1b98-47ff-8bec-62e8a5534a15.png">

**So for real, what does that option even do?**

**Comment by ni-max** [comment]
I have the same issue with 10.13.1, Is it a bug?

**Comment by essalkniro** [comment]
Its not a Bug. There must be more then 10 Items and the search and filtering will appear.
